20181204-GARY.java

```diff
@@ -1,6 +1,7 @@
 public class Hiddenword{
 
 public String HW；
+
 public String Guess；
 
 public Hiddenword(String HW){
```

Comment:
**Line 5**
No need declare an variable called *Guess*, it will be a parameter for method getHint(String guess)
**Line 13**
Use str.substring(i, i + 1) for getting a single character from a string.
**Line 14, 17, 20**
s1.equals(s2), **not equalTo()**